src/msw/menuitem.cpp

```diff
@@ -972,6 +973,19 @@ bool wxMenuItem::OnDrawItem(wxDC& dc, const wxRect& rc,
             }
 
             hTheme.DrawBackground(hdc, rcSelection, MENU_POPUPITEM, state);
+            // we need also to draw menu arrow if the menu item at popup menu and has subMenu for dark mode.
```

Comment:
If I understand the meaning of this comment correctly, it should say
```suggestion
            // we need also to draw menu arrow of sub-menus in dark mode
```